packages/cms/pages/auth/register/register.tsx

```diff
@@ -12,12 +12,30 @@ import { EuiFlexGroup } from '@tensei/eui/lib/components/flex'
 import { EuiFieldText } from '@tensei/eui/lib/components/form/field_text'
 import { EuiFieldPassword } from '@tensei/eui/lib/components/form/field_password'
 import { EuiFlexItem } from '@tensei/eui/lib/components/flex/flex_item'
+import { useState } from 'react'
 
 const H3 = styled.h3`
   text-align: center;
 `
 
 export const Register: React.FunctionComponent = () => {
+  const [user, setUser] = useState({
+    firstname: '',
+    lastname: '',
+    email: '',
+    password: ''
+  })
+
+  const submitRegistration = async (e: any) => {
+    try {
+      e.preventDefault()
+      let resp = await window.Tensei.client.post('users', user)
```

Comment:
instead of `resp`, let's call it `response`. Also, you can call `event.preventDefault()` outside of the try/catch. Also here, the endpoint is `/auth/register`.

Reply:
Thanks for the endpoint.